api/tests/unit/environments/test_unit_environments_views.py

```diff
@@ -0,0 +1,47 @@
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from environments.models import Environment
+
+
+def test_retrieve_environment(
```

Comment:
Should this test be named `test_retrieve_environment_details` for clarity?

Reply:
I guess it could be, yes. I would argue that it's in a file called `test_unit_environments_views` and `retrieve` is the name of the view action we're testing, so it's not necessary to add `_details` here.